### DISCUSSION TITLE
Language Card is not Live

### DISCUSSION THREAD

**Discussion by asad-centrox** [anuraghazra/github-readme-stats#3793] [Q&A] [answered]
### Describe the bug

![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=()my_user_name&layout=compact)

this markdown adds the space but not showing any languages in it.
![LD](https://github.com/anuraghazra/github-readme-stats/assets/150003006/83570ea3-e28b-4605-b93c-8897a6d8e733)

**Comment by qwerty541** [comment, marked as answer]
Hey @asad-centrox. This is expected behavior since there is actually no data to show. You have just one repository without any languages. If you are talking about private repositories, then our documentation clearly says that by default stats card shows the data from public repositories only. To show the data from private repositories you have to [deploy your own Vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) using your own GitHub API token.
